application_form/api/sales/views.py

```diff
@@ -116,16 +117,25 @@ def sold_apartments(request):
             f"Start date {start_time_obj} cannot be greater than "
             f"end date {end_time_obj}"
         )
+
+    # Select the latest state event of every apartment uuid
     state_events = (
         ApartmentReservationStateChangeEvent.objects.filter(
             timestamp__range=[start_time_obj, end_time_obj],
-            state=ApartmentReservationState.SOLD,
         )
-        .select_related("reservation")
```

Comment:
Was this removed intentionally? Seems like it could still help.

Reply:
Seems to be an accident, I reverted it back